src/hotspot/share/opto/vectorization.hpp

```diff
@@ -111,6 +112,22 @@ class VLoop : public StackObj {
     return vtrace().is_trace(TraceAutoVectorizationTag::PRECONDITIONS);
   }
 
+  bool is_trace_loop_analyzer() const {
+    return vtrace().is_trace(TraceAutoVectorizationTag::LOOP_ANALYZER);
```

Comment:
I think this is a bit of a style question. I prefer these accessors, because it makes refactoring easier. You can just replace a field by a call to some other method later. And I make sure to have all the accessors be `const`. So side-effects should really not be happening, unless someone hacks the `const` away...

Reply:
I am now using `_vtrace`.